staging/src/k8s.io/apiserver/pkg/admission/plugin/cel/validator.go

```diff
@@ -308,3 +312,11 @@ func createAdmissionRequest(attr admission.Attributes) *admissionv1.AdmissionReq
 		},
 	}
 }
+
+func setDecisionKindForError(decision *policyDecision, failurePolicy v1alpha1.FailurePolicyType) {
+	kind := policyDecisionKindForError(failurePolicy)
+	if kind == admit {
+		decision.denyIgnored = true
```

Comment:
Yeah, I don't understand what this means, can you clarify the name and/or add a comment?

Reply:
(marking as resolved since `denyIgnored` has been removed)